src/store/messagesStore.js

```diff
@@ -1410,6 +1410,10 @@ const actions = {
 	async easeMessageList(context, { token }) {
 		context.commit('easeMessageList', { token })
 	},
+
+	loadedMessagesOfConversation(context, { token }) {
```

Comment:
Good addition! Please check if we need to clean it up from the store somewhere (like on deleteConversation)

Reply:
Nah, it is used to mark the first loading of a conversation so it should be kept there until page reload.